xbmc/filesystem/CurlFile.cpp

```diff
@@ -829,8 +829,29 @@ void CCurlFile::ParseAndCorrectUrl(CURL &url2)
           m_postdata = Base64::Decode(value);
           m_postdataset = true;
         }
-        else
+        // other standard headers (see https://en.wikipedia.org/wiki/List_of_HTTP_header_fields)
```

Comment:
I am actually limiting the number of headers here. It was initially just appending every other parameter:

`SetRequestHeader(it->first, value);`